src/spell-kind/spells-sight.cpp

```diff
@@ -370,8 +370,9 @@ bool deathray_monsters(PlayerType *player_ptr)
  * @param player_ptr プレイヤー情報への参照ポインタ
  * @param m_ptr モンスター情報への参照ポインタ
  * @param r_ptr モンスター種族への参照ポインタ
+ * @return std::string プローブからの印刷可能な結果
```

Comment:
It's natural Japanese sentense:
```
@return 調査結果 善悪アライメント、最大HP、残りHP、AC、速度、ステータス
```

Meaning:
The result of probing: alignment, max HP, current HP, AC, speed, statuses